Code/Framework/AzFramework/CMakeLists.txt

```diff
@@ -25,13 +26,20 @@ ly_add_target(
             ${pal_dir}
         PRIVATE
             ${common_dir}
+            AZ::AzNetworking
     BUILD_DEPENDENCIES
         PRIVATE
             AZ::AzCore
         PUBLIC
-            AZ::GridMate
+            AZ::AzNetworking
```

Comment:
Is this a circular dependency?

Also why should AzFramework depend on AzNetworking?

Reply:
The same reason it previously depended on GridMate. TargetManagement is part of AzFramework and requires transport to work.